core/templates/pages/topics-and-skills-dashboard-page/modals/unassign-skill-from-topics-modal.component.ts

```diff
@@ -56,45 +57,53 @@ export class UnassignSkillFromTopicsModalComponent
     super(ngbActiveModal);
   }
 
+  fetchTopicIdToDiagnosticTestSkillIds(
+      topicAssignments: AssignedSkill[]): void {
+    let topicIdsNotEligibleForUnassignment: string[] = [];
```

Comment:
I think I suggested before reframing the variable to topicIdsEligibleForUnassignment or something without a double negative. You replied "Updated" to that comment, but this still seems to have a double negative?

Reference: https://github.com/oppia/oppia/pull/15742#discussion_r936424665

Reply:
Removed the use of `topicIdsNotEligibleForUnassignment`.

Now I have divided the topicAssignment into two parts -- `eligibleTopicAssginments` and `nonEligibleTopicAssignments`, PTAL.